logic/transaction_pool.js

```diff
@@ -975,13 +990,34 @@ __private.applyUnconfirmedList = function(transactions, cb, tx) {
  */
 __private.transactionTimeOut = function(transaction) {
 	if (transaction.type === transactionTypes.MULTI) {
-		return transaction.asset.multisignature.lifetime * 3600;
+		return (
+			transaction.asset.multisignature.lifetime *
+			library.config.transactions.multiSignatureTimeoutMultiplier
+		);
 	} else if (Array.isArray(transaction.signatures)) {
-		return constants.unconfirmedTransactionTimeOut * 8;
+		return (
+			constants.unconfirmedTransactionTimeOut *
+			library.config.transactions.signaturesTimeoutMultiplier
```

Comment:
Again its confusing here, why 8 times of actual transaction timeout? Please add some comments to explain it.

Reply:
Honestly, I am not sure why 8 times, if anybody knows why I am happy to update the same in the comments.